Src/Swagger/ValidationSchemaProcessor.cs

```diff
@@ -67,15 +68,9 @@ public void Process(SchemaProcessorContext context)
         if (validatorTypes?.Length is null or 0)
             return;
 
-        if (context is null)
-        {
-            _logger?.LogError("SchemaProcessorContext is null");
-            return;
-        }
-
```

Comment:
According to nullable reference type annotations this code is not necessary, so i cleaned up a bit..